### PR TITLE
add xml.sax._exceptions module

### PR DESCRIPTION
**Pull request by tungol**
related to https://github.com/python/typeshed/issues/3968

### CODE REVIEW

File: stdlib/xml/sax/_exceptions.pyi [hunk 1]
```diff
@@ -0,0 +1,19 @@
+from typing import Any, NoReturn
+from xml.sax.xmlreader import Locator
+
+class SAXException(Exception):
+    def __init__(self, msg: str, exception: Exception | None = None) -> None: ...
+    def getMessage(self) -> str: ...
+    def getException(self) -> Exception: ...
+    def __getitem__(self, ix: Any) -> NoReturn: ...
```
Comment (srittau) [inline stdlib/xml/sax/_exceptions.pyi:8]:
```suggestion
    def __getitem__(self, ix: object) -> NoReturn: ...
```

File: stdlib/xml/sax/_exceptions.pyi [hunk 2]
```diff
@@ -0,0 +1,19 @@
+from typing import Any, NoReturn
```
Comment (srittau) [inline stdlib/xml/sax/_exceptions.pyi:1]:
```suggestion
from typing import NoReturn
```

**Review by srittau — APPROVED** [review]
LGTM, but while we're here we could update one annotation, see below.

**Comment by tungol** [comment]
Makes sense to me

**Pull request merged**